backend/control_center/notification/ui_serializers.py

```
@@ -0,0 +1,28 @@
+        model = Notification
+        fields = ["id", "read", "type", "description", "user", "urgency", "created_at"]
+
+    def get_type(self, obj: Notification) -> str:
+        # Default type until categorization is added
+        return "OTHER"
+
+    def get_user(self, obj: Notification):
+        # Optional surface of notifier information if available
+        try:
+            return obj.notifier.chat_id if obj.notifier else None
+        except Exception:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)